sklearn/decomposition/online_lda.py

```diff
@@ -305,7 +305,8 @@ def _init_latent_vars(self, n_features):
         """Initialize latent variables."""
 
         self.random_state_ = check_random_state(self.random_state)
-        self.n_iter_ = 1
+        self.weight_update_counter_ = 1
+        self.n_iter_ = 0
```

Comment:
Yes It is the number of passes over the dataset. 
Added information about the n_iter_ variable.